CooperationHullMainSite/wwwroot/css/site.css

```diff
@@ -47,8 +47,8 @@ body {
     font-optical-sizing: auto;
     font-weight: 600;
     line-height: 1.7;
-    max-width: 1920px;
-    margin: auto;
+    /*    max-width: 1920px;
```

Comment:
If we're sure we don't need this, it might be worth removing? Or at least adding a comment explaining why its commented out